### ISSUE TITLE
Old "vnu.jar.zip" in the latest release

### ISSUE THREAD

**Issue by tenzap** [validator/validator#2094]
### Summary

I just downloaded the vnu.jar.zip and found that the jar inside it reports 

$ java -jar vnu.jar --version
26.02.10 (5d98c5e)

Then I noticed that in the assets, vnu.jar.zip* is actually dated 10/Feb/2026

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
n/a
```

### Checker error messages

```shell
n/a
```

### Steps to reproduce

1.n/a
2.
3.

### Expected behavior

n/a

### Actual behavior

n/a
